help/destinations/catalog/adobe/marketo-engage-person-sync.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Marketo Engage Person Sync
+description: Use the Marketo Engage Person Sync connector to stream updates from a person audience to the corresponding records in your Marketo Engage.
+---
+
+# Marketo Engage Person Sync connection {#marketo-engage-person-sync}
+
+## Overview {#overview}
+
+The Marketo Engage Person Sync Connector enables Marketers to stream updates from an AEP person audiences to the corresponding records in your Marketo Engage instance.
+
+## Supported Identities and Attributes {#support-identities-and-attributes}
```

Comment:
before going into this part, let's talk about:

1. any prerequisites that need to be met in Marketo? 
2. how this destination is different to the existing Marketo destination
3. any callouts from this internal Slack thread: https://adobedx.slack.com/archives/C5RMQUHB9/p1730396494873629